src/client/content_mapblock.cpp

```diff
@@ -574,7 +574,8 @@ void MapblockMeshGenerator::drawLiquidSides()
 			pos.X = (base.X - 0.5f) * BS;
 			pos.Z = (base.Z - 0.5f) * BS;
 			if (vertex.v) {
-				pos.Y = neighbor.is_same_liquid ? corner_levels[base.Z][base.X] : -0.5f * BS;
+				pos.Y = neighbor.is_same_liquid ? corner_levels[base.Z][base.X] :
+												  -0.5f * BS;
```

Comment:
CF seems to have a bitter relationship with ternary operators, when it comes to indentation...